catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/FormulaEditorEditTextTest.java

```diff
@@ -0,0 +1,393 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2017 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.uiespresso.content.brick.app;
```

Comment:
please move this test to the formulaEditor package.

Reply:
Ok sure. Will try to fix these issues ASAP.
Have you already taken a look at my question?
 (https://github.com/Catrobat/Catroid/pull/2724#issuecomment-368007494)
I already tried several things but none of them worked. I really need some help with that issue I tried everything I could come up with by now... Would you mind taking a look into this together some time next week in the panic room?